### PR TITLE
Bugfix account name and disable register button

### LINKED ISSUE #5352: Account name display discrepancy
### Expected behavior

Should display account name for all types of accounts

### Actual behavior

- Account name disappears in wallet details page when opening account details modal
- Account modal do not display name when its multisig or hardware wallet account

### Steps to reproduce

Screenshot 2023-09-30 at 11 27 06 AM

Screenshot 2023-09-30 at 11 27 17 AM

### Which version(s) does this affect? (Environment, OS, etc...)
3.0.0-rc.1

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #5352

### How was it solved?

- Show valid account name
- Disable register button until evaluated 

### How was it tested?

- Check account name against current or searched account, use issue for verification
- Register validator button should be disabled while its evaluating the state

### CODE REVIEW

File: src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js [hunk 1]
```diff
@@ -173,7 +178,11 @@ const ValidatorsMonitor = ({ watchList }) => {
                 <SecondaryButton>{t('Stakes')}</SecondaryButton>
               </Link>
             )}
-            <ValidatorActionButton isValidator={isValidator} address={address} />
+            <ValidatorActionButton
+              isValidator={isValidator}
+              isLoadingValidators
```
Comment (ikem-legend) [inline src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js]: This prop isn't among the props for the component and hence has no effect. Was the intended change `isValidator={isValidator || isLoadingValidators}`?

File: src/modules/wallet/components/AccountDetails/index.js [hunk 2]
```diff
@@ -178,12 +181,10 @@ const AccountDetails = () => {
...
               {readMode ? (
                 <>
-                  <h3>{queryAddress ? authData?.meta?.name : currentAccount.metadata?.name}</h3>
+                  <h3>{accountName}</h3>
                   {numberOfSignatures > 0 && <Icon name="multisigKeys" />}
-                  {!authLoading && !queryAddress && (
-                    <Icon name="edit" onClick={(e) => setMode(e)} />
-                  )}
-                  {!queryAddress && currentAccount.metadata.isHW && (
+                  {!authLoading && isMyAccount && <Icon name="edit" onClick={(e) => setMode(e)} />}
```
Comment (eniolam1000752) [inline src/modules/wallet/components/AccountDetails/index.js]:
```suggestion
                  {!authLoading && isMyAccount && <Icon name="edit" onClick={setMode} />}
```

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

**Review by eniolam1000752 — DISMISSED** [review]

File: src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js [hunk 3]
```diff
@@ -173,7 +178,11 @@ const ValidatorsMonitor = ({ watchList }) => {
                 <SecondaryButton>{t('Stakes')}</SecondaryButton>
               </Link>
             )}
-            <ValidatorActionButton isValidator={isValidator} address={address} />
+            <ValidatorActionButton
+              isValidator={isValidator}
+              isLoadingValidators={isLoadingValidators}
+              address={address}
```
Comment (ikem-legend) [inline src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js]: `ValidatorActionButton` takes only two props. Therefore `isLoadingValidators` can either be added to its definition or used differently here

Comment (ManuGowda) [reply src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js]: This prop was not necessary as its handled via isValidator, so removed.

**Review by ikem-legend — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**